frontend/src/lib/modals/sns/neurons/FollowSnsNeuronsByTopicItem.svelte

```diff
@@ -46,6 +48,7 @@
 
   let toggleContent: () => void = $state(() => {});
   let expanded: boolean = $state(false);
+  let isFollowingByTopic = $derived(followees.length > 0);
```

Comment:
this could be a `const` as the value doesn't change in the component, no?
```suggestion
  const isFollowingByTopic = $derived(followees.length > 0);
```